.github/workflows/pr.yml

```diff
@@ -7,7 +7,7 @@ name: Build pr
 
 on:
   pull_request:
-    branches: [main, release/**, next/**, feat/add-jest-tests]
```

Comment:
How about we just remove the branch filter altogether so that every PR can have the CI flow run?

Probably want to do the same in the `pr-storybook.yml` file too so that we can get storybook builds on any PR we want